micrometer-observation/src/main/java/io/micrometer/observation/Observations.java

```diff
@@ -36,7 +36,6 @@ public final class Observations {
             initialRegistry);
 
     private Observations() {
-        throw new UnsupportedOperationException("You can't instantiate a utility class");
```

Comment:
Was this causing a test failure?

Reply:
@shakuzen No, I just removed it as it doesn't seem to be necessary.

Reply:
Fair enough. I agree.